docs/src/pages/components/drawers/ResponsiveDrawer.js

```diff
@@ -86,8 +84,10 @@ function ResponsiveDrawer(props) {
     </div>
   );
 
+  const [container, setContainer] = React.useState(null);
+
   return (
-    <div className={classes.root}>
+    <div className={classes.root} ref={setContainer}>
```

Comment:
I think that we should optimize for the developers that drop the demo into their project. Shouldn't the body be the container, not this div?

Reply:
> I think that we should optimize for the developers that drop the demo into their project.

We didn't before, but sure let's go with the ownerDocument of this div.

Reply:
Honestly I'd like to just get rid of iframes. They make things just so much more complicated since you have to rethink where `window` points to. It's tempting to use a magic `useWindow` hook that uses a Context under the hood.

In the attempt to make the demos look closer to how they would look in the actual app they're actually more complicated.

Reply:
That being said, I find `window` a lot better than `container`. `window` is something that works outside of our docs whereas it's unknown where `container` points to.

Reply:
Regarding the use cases for `"iframe": true` in the demos. We use it for the components that interact with the top-level: Drawer, AppBar, Container. I believe we will have more components in the future to address this concern: Top nav bar, [Result component](https://trello.com/c/WMO9l8Fo/2321-results-component), [Layout component](https://trello.com/c/zvQJKsyg/1580-layout-component). I'm not aware of any good alternative to the iframe.